components/blitz/src/ome/services/blitz/repo/ManagedImportRequestI.java

```diff
@@ -261,6 +275,41 @@ public void init(Helper helper) {
         }
     }
 
+    /**
+     * If the {@link #setResources()} has been called with a non-null
```

Comment:
From https://ci.openmicroscopy.org/view/DEV/job/OMERO-DEV-merge-build/233/warnings39Result/

```
Tag @link: can't find setResources() in ome.services.blitz.repo.ManagedImportRequestI
```

`#setResources(Resources)` instead should fix this